providers/oracle/src/airflow/providers/oracle/hooks/oracle.py

```diff
@@ -144,7 +144,7 @@ def get_conn(self) -> oracledb.Connection:
 
 
         """
-        conn = self.get_connection(self.oracle_conn_id)  # type: ignore[attr-defined]
+        conn = self.get_connection(self.oracle_conn_id)
```

Comment:
@gopidesupavan +1 for https://github.com/apache/airflow/pull/53252#discussion_r2202912638